src/main/scala/firrtl/stage/FirrtlAnnotations.scala

```diff
@@ -64,6 +64,41 @@ object FirrtlFileAnnotation extends HasShellOptions {
 
 }
 
+/** Read a directory of FIRRTL files or ProtoBufs
+  *  - set with `-I/--input-directory`
+  * @param file input filename
```

Comment:
this `@param` needs updating, shouldn't this be `dir`?

Reply:
also unclear the comment a little -- so the same argument `-I` can be used for a directory of both firrtl files or protobuf files? can both be in the same directory?